core/src/trezor/messages.py

```
@@ -58,6 +58,8 @@ def __getattr__(name: str) -> Any:
     from trezor.enums import ResourceType  # noqa: F401
     from trezor.enums import SafetyCheckLevel  # noqa: F401
     from trezor.enums import SdProtectOperationType  # noqa: F401
+    from trezor.enums import SolanaMessageFormat  # noqa: F401
```

Ruff (0.8.2)

Unused `noqa` directive (unused: `F401`)

Remove unused `noqa` directive

(RUF100)